ZenPacks/zenoss/WBEM/patches.py

```diff
@@ -21,7 +21,9 @@
     EnumerateClassNames as BaseEnumerateClassNames,
     EnumerateClasses as BaseEnumerateClasses,
     EnumerateInstanceNames as BaseEnumerateInstanceNames,
-    EnumerateInstances as BaseEnumerateInstances
+    EnumerateInstances as BaseEnumerateInstances,
+    OpenEnumerateInstances as BaseOpenEnumerateInstances,
+    PullInstances as BasePullInstances,
 )
```

Comment:
Just as a proposal. Maybe it would be better to use `twisted_client.Enumerate...` instead of importing all these classes with Base prefix, since twisted_client is already imported at line 14.
Or it can be even imported with shortcut, like this: `from pywbem import twisted_client as client` or something like this.